explainer.md

```diff
@@ -1489,19 +1489,22 @@ const settingsBucket = await navigator.storageBuckets.openOrCreate("settings", {
 const emailsPerPage = settingsBucket.localStorage.getItem('emailsPerPage');
 ```
 
-This alternative was rejected because we were concerned about the performance
-implications of supporting multiple `localStorage` instances per origin.
-
-Due to the synchronous nature of the Web Storage API, user agents implement
-`localStorage` via a full in-memory cache.
-
-TODO: Explain why we chose not to integrate `localStorage`.
-
-* LocalStorage doesn't follow durability policies. Sync write API means we need
-  in-renderer caches, so we couldn't support either `"relaxed"` or `"strict"`
-* RAM consumption limitation. Would need special quota system just for
-  LocalStorage. Buckets with small `quota` would get access then.
-* Non-issue: sync read API. We can read all data on `openOrCreate()`.
+This alternative was rejected because of `localStorage` does not follow
+the same durability policies, and we would need to create a quota system
+just to support it.
+
+Due to the synchronous nature of Web Storage API, user agents would need an
+in-renderer cache to support writes for `localStorage`.
+Therefore we would be able to support `"relaxed"` or `"strict"` policies.
+
+We would also need a special quota system just for `localStorage` and
```

Comment:
"We" is a bit ambiguous here.

We could be the API designers, or browser vendors, or both. Either way, API designers (standards authors) and browser vendors are below web developers in the order of constituents. So, WDYT about strengthening the  argument by explaining that developers would need to reason about this extra quota system?

Reply:
I tried changing the perspective here to the web developer perspective.